js/CornerWidgetUI.js

```diff
@@ -743,64 +812,96 @@ CornerWidgetUI._raise_lead = function (btn_ref){
 		return true;
 	}
 
-	//Limit field validation to preventing submission only X number of times
-	if ( (! jQuery("#"+CornerWidgetUI.constants._uivar_leadtelID).intlTelInput("isValidNumber")) && CornerWidgetUI._ui_config.validation.count < CornerWidgetUI._ui_config.validation.limit ){
-		jQuery("#"+CornerWidgetUI.constants._uivar_leadtelID).addClass("a-gorilla-error");
-		setTimeout(function(){
-			jQuery("#"+CornerWidgetUI.constants._uivar_leadtelID).removeClass("a-gorilla-error");
-		},1000);
-		CornerWidgetUI._ui_config.validation.count++;
-		$lucep.send_intelligence({event_type: "bad-number-entry",
-								  payload: {number: tel_no,
-											validationCount: CornerWidgetUI._ui_config.validation.count}});
+	if (window.intlTelInputUtils) {
+		//Limit field validation to preventing submission only X number of times
+		if ( (! CornerWidgetUI.jQuery("#"+CornerWidgetUI.constants._uivar_leadtelID).intlTelInput("isValidNumber")) && CornerWidgetUI._ui_config.validation.count < CornerWidgetUI._ui_config.validation.limit ) {
+			CornerWidgetUI._fail_submit(CornerWidgetUI.constants._uivar_leadtelID)
+			$lucep.send_intelligence({event_type: "bad-number-entry",
+									  payload: {number: tel_no,
+												validationCount: CornerWidgetUI._ui_config.validation.count}});
+			return false;
+		}
+	} else {
+		// if tel input validation isn't working then use 
+		if (CornerWidgetUI.jQuery("#"+CornerWidgetUI.constants._uivar_leadtelID).val().length < 4) {
```

Comment:
We need to enhance this check - if the text contains "jQuery" then there has been a failure in the text capture operation and an alternative needs to be applied